File: app/src/main/java/com/tda/app/data/model/response/OrderResponse.java
package com.tda.app.data.model.response;

import com.tda.app.data.model.Voucher;

import java.sql.Timestamp;
import java.util.List;


public class OrderResponse {
    public long orderId;
    public long userId;
    public double shippingCosts;
    public double discount;
    public double total;
    public int orderStatus;
    public int paymentMethod;
    public Timestamp createDate;
    public String address_detail;
    public long address_id;
    public Voucher voucher;
    public List<OrderItemResp> order_items;

}

File: app/src/main/java/com/tda/app/model/response/OrderResponse.kt
package com.tda.app.model.response


import java.sql.Timestamp

data class OrderResponse (
    val orderId: Long,
    val userId: Long,
    val shippingCosts:Int,
    val discount:Double,
    val total:Double,
    val orderStatus:Int,
    val paymentMethod:Int,
    val createDate: Timestamp,
    val address_detail: String,
    val address_id: Long,
   // val voucher: Voucher? = null
    val order_items: List<OrderItemResp>
)